README.md

```diff
@@ -1 +1,50 @@
-# gitops-lab-infra
```

Comment:
Folder structure should be clusters/preprod/my-cluster. Maybe an environment will have one day multiple clusters who knows

Reply:
Will do it now

Reply:
Done! Please review it, thanks.